src/test/java/com/uid2/shared/store/reader/RotatingS3KeyProviderTest.java

```diff
@@ -0,0 +1,2 @@
+package com.uid2.shared.store.reader;public class RotatingS3KeyProviderTest {
```

Comment:
Looks like you are still working on this but format it :)

Reply:
yes, this is done now!